bundles/pascal/misc/example.pas

```diff
@@ -0,0 +1,53 @@
+{ Comments }
```

Comment:
Hmm, or it seems to be partially working. I might have lucked out with a weird example here: [test file](https://github.com/graemeg/freepascal/blob/master/compiler/aoptobj.pas). When I open this the first comment isn't highlighted correctly, but any edits, etc. makes it work. A similar file seems to work out fine.